clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -436,6 +445,10 @@ def deploy_model(self,
             The number of replicas of the model to create. The number of replicas
             for a model can be changed at any time with
             :py:meth:`clipper.ClipperConnection.set_num_replicas`.
+        batch_size : int, optional
+            The user defined batch size.
+            The default value is -1, which means that the user did not define a specific batch size.
```

Comment:
Update the documentation for this method in the same way as for `build_and_deploy_model`.